src/styles/index.css

```diff
@@ -9,3 +11,15 @@
 :where(:not(h1,h2,h3,h4,h5,h6)) {
   font-family: var(--font-base, serif);
 }
+
+body {
+  background-color: var(--bg-color);
+  color: var(--text-color);
+}
+
+button {
```

Comment:
I don't know if we should leave it as a component instead of putting the styles to the button tag.

Reply:
Removed Styles